### PR TITLE
Fix test case resgroup_views

### PR DESCRIPTION
**Pull request by dreamedcheng**
Make test case 'resgroup_views' environmental independent.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: src/test/isolation2/output/resgroup/resgroup_views.source [hunk 1]
```diff
@@ -34,30 +34,33 @@ select * from gp_toolkit.gp_resgroup_role where rrrolname='gpadmin';
...
  groupid | groupname   | segment_id | vmem_usage 
 ---------+-------------+------------+------------
- 6438    | admin_group | -1         | 11         
- 6438    | admin_group | 0          | 8          
- 6438    | admin_group | 1          | 8          
- 6438    | admin_group | 2          | 8          
+ 6438    | admin_group | -1         | 16         
+ 6438    | admin_group | 0          | 12         
+ 6438    | admin_group | 1          | 12         
+ 6438    | admin_group | 2          | 12
```
Comment (soumyadeep2007) [inline src/test/isolation2/output/resgroup/resgroup_views.source:64]: I'm not sure I understand these diffs. Is it possible that you had a stray resource group on your system, when you ran these tests?

Comment (dreamedcheng) [reply src/test/isolation2/output/resgroup/resgroup_views.source]: I dropped the unexpected `rg_test` group in the testbed, and re-run the test. I think for the `vmem_usage` and `cpu_usage` the diff is under expectation in different run. The original test already included these statements in ignore region.

Comment (soumyadeep2007) [reply src/test/isolation2/output/resgroup/resgroup_views.source]: Thank you.

> I think for the vmem_usage and cpu_usage the diff is under expectation in different run. The original test already included these statements in ignore region.

Typically, we don't check in changes to ignored hunks. That way this commit will be cleaner and more concise. Could you revert these irrelevant diffs?

Comment (dreamedcheng) [reply src/test/isolation2/output/resgroup/resgroup_views.source]: Good suggestion, done!

**Review by soumyadeep2007 — APPROVED** [review]

**Pull request merged**